Menu/src/Option.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;
+public class Option {
+	Scanner sc = new Scanner(System.in);
+	public void mainMenu() {
+		int select;
+		System.out.println("Welcome To The Mocha Restaurant");
+		System.out.println("\n Select From The Following Options According To Your Desire");
+		System.out.println("\n Let Us Know What Would You Like To have");
+		System.out.println("==========================================");
+		System.out.println("| 1. Cold Drinks |");
+		System.out.println("| 2. Coffee |");
+		System.out.println("| 3. Tea |");
+		System.out.println("| 4. Shakes |");		
+		System.out.println("| 5. Pizza |");
+		System.out.println("| 6. Sandwitch |");
+		
+		System.out.println("===========================================");
+		System.out.println("Please Select The Option Now");
+		
+		select = sc.nextInt();
+		
+		switch(select) {
+		case 1:
+			coldDrinks();
+			break;
+		case 2:
+			coffee();
+			break;
+		case 3:
+			tea();
+			break;
+		case 4:
+			shakes();
+			break;
+		case 5:
+			pizza();
+			break;
+		case 6:
+			sandwitch();
+			break;
+		case 7:
+			System.out.println("Thank you for visiting");
+			break;
+			
+		}
+	}
+public void coldDrinks() {
+	//int select1;
+	System.out.println("You have selected cold drings.");
+	System.out.println("Here are the available cold drinks");
+	System.out.println("| 1. Mojito Mocktail |");
+	System.out.println("| 2. Coca cola |");
+	System.out.println("| 3. Lemonade |");
+	System.out.println("| 4. Nectar |");
+}

```

Comment:
It is best practice to have documentation comment. Please do it for each method.